### PR TITLE
[TEP005][QOL] Add MontecarloRunner.spectrum_integrated

### PR DESCRIPTION
**Pull request by yeganer**
This is a shortcut for obtaining an integrated spectrum for the
wavelength grid defined in the configuration file. If incompatible
configuration settings are detected, an IntegrationError is raised.

### CODE REVIEW

File: tardis/montecarlo/base.py [hunk 1]
```diff
@@ -125,6 +125,12 @@ def spectrum_virtual(self):
                 self.montecarlo_virtual_luminosity)
 
     @property
+    def spectrum_integrated(self):
+        return self.integrator.calculate_spectrum(
+                self.spectrum_frequency[:-1],
+                1000)
```
Comment (unoebauer) [inline tardis/montecarlo/base.py]: The hardcoded 1000 is the resolution of the p-grid, right? If so, I think we should make this also a configuration variable.

Comment (yeganer) [inline tardis/montecarlo/base.py]: What name would you suggest for such a configuration option?

**Comment by wkerzendorf** [comment]
@unoebauer I agree that it would be nice to have a config option for the p-values, so I would say that we put this as an item in the project todo - but that we merge this PR.

File: tardis/montecarlo/base.py (hunk 1 above)
Comment (unoebauer) [inline tardis/montecarlo/base.py]: something like ``impact_parameter_bins``

**Pull request merged**